lib/Jit/utility.cpp

```diff
@@ -43,8 +43,13 @@ bool MethodSet::contains(const char *Name, const char *ClassName,
   return false;
 }
 
-std::unique_ptr<std::string> Convert::wideToUtf8(const wchar_t *WideStr) {
-  ArrayRef<char> SrcBytes((const char *)WideStr, (2 * wcslen(WideStr)));
+std::unique_ptr<std::string> Convert::utf16ToUtf8(const char16_t *WideStr) {
+  // Get the length of the input
+  size_t SrcLen = 0;
+  for (; WideStr[SrcLen] != (char16_t)0; SrcLen++)
+    ;
```

Comment:
I can see the push back on wcslen, but is there something in the stdlib that does the thing above.  Something in me is rebelling at having this raw loop here.